### DISCUSSION TITLE
Laravel livewire Pagination crash on search time please resolve this issue i cant find way to fix this bug kindly help me for this issue.

### DISCUSSION THREAD

**Discussion by wasitmirani** [livewire/livewire#1767] [4. General]
http://livewiredesigniconic.test/livewire/message/user.users?page=4

Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException
The GET method is not supported for this route. Supported methods: POST.
http://livewiredesigniconic.test/livewire/message/user.users?page=4

**Comment by detygon** [comment]
Hi, can you please share an example repo or a snippet of your component and the route file ? It will be easier to debug.
